<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordcloudMainWindow</class>
 <widget class="QMainWindow" name="WordcloudMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>798</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>워드 클라우드</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../common/resources/app_icon.png</normaloff>../common/resources/app_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralwidget_grid">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="main_hlayout">
      <item>
       <layout class="QVBoxLayout" name="main_vlayout">
        <item>
         <widget class="QGroupBox" name="file_grp">
          <property name="title">
           <string>파일</string>
          </property>
          <layout class="QVBoxLayout" name="file_grp_vlayout">
           <item>
            <layout class="QHBoxLayout" name="file_csv_file_hlayout">
             <item>
              <widget class="QLabel" name="file_csv_file_lbl">
               <property name="text">
                <string>데이터 파일</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="file_csv_file_ledit"/>
             </item>
             <item>
              <widget class="QPushButton" name="file_csv_file_btn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="file_csv_file_sep_cmbbox"/>
             </item>
             <item>
              <widget class="QComboBox" name="file_csv_file_encoding_cmbbox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="file_font_file_hlayout">
             <item>
              <widget class="QLabel" name="file_font_file_lbl">
               <property name="text">
                <string>글꼴 파일</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="file_font_file_ledit"/>
             </item>
             <item>
              <widget class="QPushButton" name="file_font_file_btn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="file_font_file_warn_lbl">
             <property name="text">
              <string>주의: 한국어는 꼭 글꼴 파일을 선택하세요</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="stopwords_grp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>불용어 설정</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="stopwords_opt_hlayout">
             <item>
              <widget class="QCheckBox" name="stopwords_opt_chkbox">
               <property name="text">
                <string>불용어를 설정합니다</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="stopwords_hlayout">
             <item>
              <widget class="QLabel" name="stopwords_lbl">
               <property name="text">
                <string>불용어</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="stopwords_ledit"/>
             </item>
             <item>
              <widget class="QPushButton" name="stopwords_btn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="stopwords_warn_lbl">
             <property name="text">
              <string>예시: 사과,바나나,키위 (쉼표로 구분)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="opt_grp">
        <property name="title">
         <string>옵션</string>
        </property>
        <layout class="QVBoxLayout" name="opt_grp_vlayout">
         <item>
          <layout class="QHBoxLayout" name="opt_width_hlayout">
           <item>
            <widget class="QLabel" name="opt_width_lbl">
             <property name="text">
              <string>너비</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="opt_width_sbox">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>800</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="opt_height_hlayout">
           <item>
            <widget class="QLabel" name="opt_height_lbl">
             <property name="text">
              <string>높이</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="opt_height_sbox">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>600</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="opt_minfont_hlayout">
           <item>
            <widget class="QLabel" name="opt_minfont_lbl">
             <property name="text">
              <string>최소 글꼴 크기</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="opt_minfont_sbox">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="opt_maxfont_hlayout">
           <item>
            <widget class="QLabel" name="opt_maxfont_lbl">
             <property name="text">
              <string>최대 글꼴 크기</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="opt_maxfont_sbox">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>200</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="opt_background_hlayout">
           <item>
            <widget class="QCheckBox" name="opt_background_chkbox">
             <property name="text">
              <string>배경 이미지를 설정합니다</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="opt_background_file_hlayout">
           <item>
            <widget class="QLabel" name="opt_background_file_lbl">
             <property name="text">
              <string>배경 이미지 파일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="opt_background_file_ledit"/>
           </item>
           <item>
            <widget class="QPushButton" name="opt_background_file_btn">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="result_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="preview_tblview"/>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="wordcloud_canvas"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="btn_grp">
      <item>
       <widget class="QPushButton" name="view_preview_table_btn">
        <property name="text">
         <string>데이터 보기 (&amp;D)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="make_wordcloud_btn">
        <property name="text">
         <string>워드 클라우드 만들기 (&amp;M)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_wordcloud_btn">
        <property name="text">
         <string>워드 클라우드 저장 (&amp;S)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionqnfdydd">
   <property name="text">
    <string>불용어 제거</string>
   </property>
  </action>
  <action name="actionq">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
